.github/workflows/self-past.yml

```diff
@@ -15,6 +15,11 @@ on:
       version:
         required: true
         type: string
+      # Use this to control the commit to test against
```

Comment:
So we can have a control to test against a targe commit. For example, (Patch release: v4.21.2)